src/manager/job.c

```diff
@@ -0,0 +1,80 @@
+#include "job.h"
+#include "manager.h"
+#include "node.h"
+
+static const sd_bus_vtable job_vtable[] = { SD_BUS_VTABLE_START(0), SD_BUS_VTABLE_END };
+
+
+Job *job_new(Node *node, const char *unit, const char *type) {
+        static uint32_t next_id = 0;
+
+        _cleanup_job_ Job *job = malloc0(sizeof(Job));
+        if (job == NULL) {
+                return NULL;
+        }
+
+        job->ref_count = 1;
+        job->state = JOB_WAITING;
+        job->id = ++next_id;
```

Comment:
This can just be set to 1.

Reply:
What do you mean one? We need a unique id for each job.

Reply:
Ah, I missed ```next_id``` is static.